Automattic/vip-go-mu-plugins
# Update wp-memcached submodule.

## Description

The object-cache drop in hasn't been updated for a while and there's
been an important fix for the `$found` parameter added.

This is particularly important for some plugins, e.g. MLP3 depends on this functionality and performance declines without it.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Install MLP3
2. Notice a number of queries constantly being run on the homepage
3. Update the submodule to this PR
4. The queries should now be gone as the cache should be getting hit.
